doc/02_user/examples.rst

```diff
@@ -3,16 +3,25 @@
 Examples
 ========
 
-To make use of Ibex it has to be integrated as described in :ref:`core-integration`.
+There are two examples that demonstrate Ibex usage.
```

Comment:
I think this commit be merged into the one removing the example from this repo.

Reply:
I see your point, but I think I'll leave them separate just to reduce the amount of rebasing. The commit updating the examples does do more than just alter the reference to the FPGA example too so they'd necessarily go neatly together.